include/mbedtls/pk.h

```diff
@@ -869,6 +877,32 @@ int mbedtls_pk_parse_key(mbedtls_pk_context *ctx,
 int mbedtls_pk_parse_public_key(mbedtls_pk_context *ctx,
                                 const unsigned char *key, size_t keylen);
 
+#if defined(MBEDTLS_USE_PSA_CRYPTO)
+/**
+ * \brief   Copy the public key content in raw format from "ctx->pk_ctx"
+ *          (which is an ecp_keypair) into the internal "ctx->pk_raw" buffer.
+ *
+ * \note    This is a temporary function that can be removed as soon as the pk
+ *          module is free from ECP_C
```

Comment:
> is there are reason this function is public?

Since it was used in tests I thought it has to be public as if tests were "standard" end user applications. However I just checked that this is not the case, so I'm going to place it in `pk_internal.h`